components/link/link.js

```diff
@@ -139,13 +156,31 @@ class Link extends FocusMixin(LitElement) {
 			'truncate': this.lines > 0
 		};
 		const styles = (this.lines > 0) ? { '-webkit-line-clamp': this.lines } : {};
+		if (this.newWindow && this.target === undefined) {
+			this.target = '_blank';
+		}
 		return html`<a
 				aria-label="${ifDefined(this.ariaLabel)}"
 				class="${classMap(linkClasses)}"
 				style="${styleMap(styles)}"
 				?download="${this.download}"
 				href="${ifDefined(this.href)}"
-				target="${ifDefined(this.target)}"><slot></slot></a>`;
+				target="${ifDefined(this.target)}"><slot></slot>${this.renderNewWindowIcon()}</a>`;
+	}
+
+	renderNewWindowIcon() {
+		const newWindowClasses = {
+			'd2l-new-window': true,
+			'd2l-icon-small': this.small,
+		};
+		return this.newWindow ? html`
+			<d2l-icon class="${classMap(newWindowClasses)}" aria-hidden="true" icon="tier1:new-window"></d2l-icon>
+			<span class="d2l-offscreen">${this._newWindowLangTerm}</span>
```

Comment:
I don't think the extra getter for the lang term is necessary.
```suggestion
			<span class="d2l-offscreen">${this.localize('components.link.open-in-new-window')}</span>
```